doc/source/user/library/examples.rst

```diff
@@ -966,3 +966,318 @@ modules while in ``u.modules_stats`` the two dimension are switched.
+	 */
+	auto polar_encode_multiple(const int k, const int n,
+				   const std::vector<bool> &frozen_bits,
+				   const std::vector<std::vector<int>> &info_bits) {
+	  // populate vector
+	  std::vector<std::vector<int>> encoded_bits;
+
+	  // encode
+	  module::Encoder_polar<int> polar_encoder(k, n, frozen_bits);
+	  for (auto frame : info_bits) {
+	    std::vector<int> encoded(n);
+	    polar_encoder.encode(frame, encoded);
+	    encoded_bits.push_back(encoded);
+	  }
+	  return encoded_bits;
+	}
+
+	/**
+	 * @brief Polar decoder for single frame
+	 *
+	 * @param k The number of information bits
+	 * @param n The codeword length
+	 * @param frozen_bits std::vector<bool>, frozen bits (length k)
+	 * @param received std::vector<float> soft symbols, BPSK, length n
+	 * @return auto Decoded information bits, std::vector<int> of length k
+	 */
+	auto polar_decode(const int k, const int n,
+			  const std::vector<bool> &frozen_bits,
+			  const std::vector<float> &received) {
+	  // populate vector
+	  std::vector<int> decoded_bits(k);
+
+	  // decode
+	  module::Decoder_polar_SC_naive<int> polar_decoder(k, n, frozen_bits);
+	  polar_decoder.decode_siho(received, decoded_bits);
+	  return decoded_bits;
+	}
+
+	/**
+	 * @brief Polar decoder for multiframe
+	 *
+	 * @param k The number of information bits
+	 * @param n The codeword length
+	 * @param frozen_bits std::vector<bool>, frozen bits (length k)
+	 * @param received std::vector<float> soft symbols, BPSK, n_frame rows, n
+	 * columns
+	 * @return auto Decoded information bits, std::vector<std::vector<int>> of
+	 * n_frame rows, k columns
+	 */
+	auto polar_decode_multiple(const int k, const int n,
+				   const std::vector<bool> &frozen_bits,
+				   const std::vector<std::vector<float>> &received) {
+	  // populate vectors
+	  std::vector<std::vector<int>> decoded_bits;
+
+	  // encode
+	  module::Decoder_polar_SC_naive<int> polar_decoder(k, n, frozen_bits);
+	  for (auto frame : received) {
+	    std::vector<int> decoded(k);
+	    polar_decoder.decode_siho(frame, decoded);
+	    decoded_bits.push_back(decoded);
+	  }
+	  return decoded_bits;
+	}
+
+Finally, :numref:`setup_py` tells the Cython how to compile our interfacing functions.
+
+.. code-block:: python
+	:caption: Setup function
+	:name: setup_py
+	:emphasize-lines: 
+	:linenos:
+	
+	from distutils.core import setup
+	from distutils.extension import Extension
+	from Cython.Build import cythonize
+
+	codec = Extension(
+	    name="codec",
+	    sources=["codec.pyx"],
+	    libraries=["aff3ct-2.3.5-384-gec40f26"],
+	    library_dirs=["../../lib/aff3ct/build/lib"],
+	    include_dirs=["../../lib/aff3ct/include", "../../lib/aff3ct/lib/cli/src", "../../lib/aff3ct/lib/MIPP/src", "../../lib/aff3ct/lib/MIPP/src", "../../lib/aff3ct/lib/rang/include"],
+	    language="c++",
+	    extra_compile_args=["-std=c++14"],
+	    extra_link_args=["-std=c++14"]
+	)
+	setup(
+	    name="codec",
+	    ext_modules=cythonize([codec])
+	)
```

Comment:
I think that we can remove this code, this is a configuration code.

Reply:
Done.